pkg/controller/servicediscovery/servicediscovery_controller.go

```diff
@@ -347,11 +347,13 @@ func newLigthhouseCoreDNSService(cr *submarinerv1alpha1.ServiceDiscovery) *corev
 	}
 }
 
-func updateDNSConfigMap(client controllerClient.Client, k8sclientSet *clientset.Clientset, cr *submarinerv1alpha1.ServiceDiscovery) error {
+func updateDNSConfigMap(client controllerClient.Client, k8sclientSet *clientset.Clientset, cr *submarinerv1alpha1.ServiceDiscovery,
+	reqLogger logr.Logger) error {
 	configMaps := k8sclientSet.CoreV1().ConfigMaps("kube-system")
 	retryErr := retry.RetryOnConflict(retry.DefaultRetry, func() error {
 		configMap, err := configMaps.Get("coredns", metav1.GetOptions{})
 		if err != nil {
+			reqLogger.Error(err, "Config map is not found")
```

Comment:
```suggestion
			reqLogger.Error(err, "Error retrieving 'coredns' ConfigMap")
```

But why log the error here where in other places below we don't. Won't the caller end up logging it?